src/containers/email-confirm-modal/EmailConfirmModal.jsx

```diff
@@ -35,6 +35,9 @@ const EmailConfirmModal = ({ confirmToken, openModal }) => {
           style={ styles }
           title={ t('modals.emailNotConfirm') }
         />
+        <Button onClick={ openLoginDialog } size='large' variant='contained'>
```

Comment:
do we need to open login if email not confirm?

Reply:
fixed